debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/databases/mysql/DockerMySqlDeployer.java

```diff
@@ -5,7 +5,7 @@
  */
 package io.debezium.testing.system.tools.databases.mysql;
 
-import static io.debezium.testing.system.tools.ConfigProperties.DOCKER_IMAGE_MYSQL;
+import static io.debezium.testing.system.tools.ConfigProperties.DOCKER_IMAGE_MYSQL_MASTER;
```

Comment:
I think we probably can avoid this naming change, adding MASTER suffix is not necessary. Out of the box if you deploy a single replica you need a master anyway. So I would keep just DOCKER_IMAGE_MYSQL and add just DOCKER_IMAGE_MYSQL_REPLICA.